src/passes/implicitConversionToExplicit.ts

```diff
@@ -294,7 +294,7 @@ function insertConversionIfNecessary(expression: Expression, targetType: TypeNod
       `Unexpected unresolved rational literal ${printNode(expression)}`,
     );
   } else if (currentType instanceof StringLiteralType) {
-    insertConversion(expression, targetType, ast);
+    return;
```

Comment:
Is something not needed here so that we can handle both string literals that become arrays and string literals that become fixedBytesTypes?

Reply:
Would it be acceptable to replace it with an int literal of the correct size at this location?